DevExtreme.AspNet.TagHelpers.Generator/IntellisenseData/Info.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace DevExtreme.AspNet.TagHelpers.Generator.IntellisenseData {
+
+    [XmlInclude(typeof(IntellisenseObjectInfo))]
+    public class Info {
+        static Info[] EmptyPropsArray = new Info[0];
+
+        ICollection<Info> _props;
+
+        [XmlAttribute]
+        public string Type;
+
+        [XmlAttribute]
+        public string Name;
+
+        [XmlAttribute]
+        public string Description;
+
+        [XmlArray("Properties")]
+        [XmlArrayItem("IntellisenseObjectPropertyInfo")]
+        public Info[] Props {
+            get { return _props?.ToArray() ?? EmptyPropsArray; }
```

Comment:
Yes, fixed in https://github.com/IlyaKhD/DevExtreme.AspNet.TagHelpers/commit/f925e164d3af09954448990703ce6522765132fc